src/main/java/com/twilio/rest/api/v2010/account/usage/Record.java

```diff
@@ -119,6 +119,10 @@ public enum Category {
         CALLS_SIP_OUTBOUND("calls-sip-outbound"),
         CALLS_TRANSFERS("calls-transfers"),
         CARRIER_LOOKUPS("carrier-lookups"),
+        CHANNELS_MESSAGING_INBOUND("channels-messaging-inbound"),
```

Comment:
This file is autogenerated from open api specs and these changes will be overridden on next release. Thus these line can not be added here.